prow/plugins.yaml

```diff
@@ -287,6 +287,15 @@ require_matching_label:
   prs: true
   regexp: ^(sig|wg|committee)/
   # Define needs-sig separately for issues so that we only comment on issues.
+- missing_label: needs-review
```

Comment:
I find this name to be a little non-obvious.  If we are going to reclaim the term "triage" to mean "sig has verified" (since some use it to mean sig-assignment) then "needs-triage" seems short and precise.  If we're not comfortable with that, then "needs-sig-review" or "needs-sig-triage" ?

Reply:
Agree on that, let's reclaim triage. I'll open a parallel issue to nuke the triage labels and/or replace them with `lifecycle` etc.

Both 'needs-sig-review' and 'needs-sig-triage' look good. Any comments @spiffxp @justaugustus @fejta  ?

Reply:
moving forward with `needs-triage` for now